### PR TITLE
[i18n] Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [HyperPlay Client/Login Page](https://hosted.weblate.org/projects/hyperplay-client/login-page/).

It also includes following components:

* [HyperPlay Client/Globals](https://hosted.weblate.org/projects/hyperplay-client/globals/)

* [HyperPlay Client/GamePage](https://hosted.weblate.org/projects/hyperplay-client/gamepage/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]

File: public/locales/es/translation.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
             },
             "moving": "Error al Mover el Juego {{error}}",
             "ubisoft-connect": {
-                "message": "La instalación de Ubisoft Connect en el prefijo del juego ha fallado. Consulta nuestra página wiki en https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/How-to-install-Ubisoft-Connect-on-Linux-and-Mac para instalarlo manualmente.",
+                "message": "La instalación de Ubisoft Connect en el prefijo del juego ha fallado. Consulte nuestra página wiki en https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/How-to-install-Ubisoft-Connect-on-Linux-and-Mac para instalarlo maunally.",
```
Comment (eliobricenov) [inline public/locales/es/translation.json:83]: this is a mistake, maunally is not a spanish word

File: public/locales/es/translation.json [hunk 2]
```diff
@@ -216,7 +217,7 @@
             "type": "Type"
         },
         "title": "Downloads",
-        "ETA": "Tiempo estimado"
+        "ETA": "Hora prevista"
```
Comment (eliobricenov) [inline public/locales/es/translation.json:220]: I'd argue the prev one was better

**Review by eliobricenov — CHANGES_REQUESTED** [review]

**Comment by eliobricenov** [comment]
@flavioislima how do we handle incorrect translations? second one I commented is fine to leave, but the first one is a mistake

**Comment by flavioislima** [comment]
> @flavioislima how do we handle incorrect translations? second one I commented is fine to leave, but the first one is a mistake

Even in Spanish from Spain they are wrong?
The correct way would be you creating an account in weblate and fixing there. Or fix it on another PR after merging this one.

**Comment by flavioislima** [comment]
@eliobricenov fixed it on weblate. Check now.

**Review by eliobricenov — APPROVED** [review]

**Pull request merged**